O/osslsigncode/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "osslsigncode"
+version = v"2.10.0"
```

Comment:
Where's v2.10.0 coming from?  In https://github.com/mtrojnar/osslsigncode/tags I only see up to v2.9

Reply:
https://github.com/mtrojnar/osslsigncode/blob/4568c890cc1538ca80be3ee36775ba42223dea04/CMakeLists.txt#L13

Reply:
I changed to a V2.9 tagged commit.